src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.targets

```diff
@@ -214,6 +214,10 @@ namespace Projects%3B
         <_Parameter1>apphostprojectpath</_Parameter1>
         <_Parameter2>$(MSBuildProjectDirectory)</_Parameter2>
       </AssemblyAttribute>
+      <AssemblyAttribute Include="System.Reflection.AssemblyMetadata">
+        <_Parameter1>apphostconfiguration</_Parameter1>
+        <_Parameter2>$(Configuration)</_Parameter2>
+      </AssemblyAttribute>
```

Comment:
```suggestion
```

We don't need to do this, the .NET SDK already does it for us:

https://github.com/dotnet/sdk/blob/cbed36adb0e24a8f3c33ba88a74f101a87410e13/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L81-L83